server_test.go

```diff
@@ -230,7 +230,7 @@ func (ts *ServerTests) TestListenAndServe() {
 
 	// Start serving
 	go func() {
-		_ = ListenAndServe(addr, queue)
+		_ = ListenAndServe(addr, queue) // #nosec G104
```

Comment:
These errors (and the gosec linter itself) don't show up on my machine using golangci-lint v1.10.2. They do when I use golangci-lint from `go get`. I don't know what's going on 🙃